src/components/TopBar/TopBar.scss

```diff
@@ -10,7 +10,7 @@ $context-control-expand-after: 1400px;
   position: relative;
   display: flex;
   height: top-bar-height();
-  background-color: var(--top-bar-background);
+  background-color: var(--surface-background, var(--top-bar-background));
```

Comment:
We might want to make this `none` instead of actually setting the background color, since `ThemeProvider` sets this as its background.